bundles/org.openhab.core.compat1x/META-INF/MANIFEST.MF

```diff
@@ -1,105 +1,110 @@
- org.eclipse.smarthome.core.audio,
- org.eclipse.smarthome.core.autoupdate,
- org.eclipse.smarthome.core.common.registry,
- org.eclipse.smarthome.core.events,
- org.eclipse.smarthome.core.items,
- org.eclipse.smarthome.core.items.events,
- org.eclipse.smarthome.core.library.items,
- org.eclipse.smarthome.core.library.types,
- org.eclipse.smarthome.core.persistence,
- org.eclipse.smarthome.core.transform,
- org.eclipse.smarthome.core.types,
- org.eclipse.smarthome.core.voice,
- org.eclipse.smarthome.model.item,
- org.eclipse.smarthome.model.persistence,
- org.eclipse.smarthome.model.script.engine,
- org.eclipse.smarthome.model.script.engine.action,
- org.eclipse.smarthome.model.sitemap,
- org.eclipse.smarthome.ui.chart,
- org.eclipse.smarthome.ui.items,
- org.eclipse.xtext.xbase,
- org.openhab.library.tel.types,
- org.osgi.framework,
- org.osgi.service.cm,
- org.osgi.service.event,
- org.osgi.service.http,
- org.osgi.util.tracker,
- org.slf4j
-Export-Package: org.codehaus.jackson,
- org.codehaus.jackson.annotate,
- org.codehaus.jackson.format,
- org.codehaus.jackson.impl,
- org.codehaus.jackson.io,
- org.codehaus.jackson.map,
- org.codehaus.jackson.map.annotate,
- org.codehaus.jackson.map.deser,
- org.codehaus.jackson.map.deser.impl,
- org.codehaus.jackson.map.deser.std,
- org.codehaus.jackson.map.exc,
- org.codehaus.jackson.map.ext,
- org.codehaus.jackson.map.introspect,
- org.codehaus.jackson.map.jsontype,
- org.codehaus.jackson.map.jsontype.impl,
- org.codehaus.jackson.map.module,
- org.codehaus.jackson.map.ser,
- org.codehaus.jackson.map.ser.impl,
- org.codehaus.jackson.map.ser.std,
- org.codehaus.jackson.map.type,
- org.codehaus.jackson.map.util,
- org.codehaus.jackson.node,
- org.codehaus.jackson.schema,
- org.codehaus.jackson.sym,
- org.codehaus.jackson.type,
- org.codehaus.jackson.util,
- org.openhab.core.autoupdate,
- org.openhab.core.binding,
- org.openhab.core.events;uses:="org.openhab.core.items,org.osgi.service.event,org.openhab.core.types",
- org.openhab.core.items;uses:="org.openhab.core.types,org.openhab.core.events",
- org.openhab.core.library.items,
- org.openhab.core.library.types,
- org.openhab.core.persistence,
- org.openhab.core.scriptengine.action,
- org.openhab.core.service,
- org.openhab.core.transform,
- org.openhab.core.transform.actions,
- org.openhab.core.types,
- org.openhab.io.console,
- org.openhab.io.multimedia.tts,
- org.openhab.io.net.actions,
- org.openhab.io.net.exec,
- org.openhab.io.net.http,
- org.openhab.library.tel.items,
- org.openhab.library.tel.types,
- org.openhab.model.item.binding,
- org.openhab.model.sitemap,
- org.openhab.ui.chart,
- org.openhab.ui.items
-Bundle-ClassPath: .,
- lib/jackson-core-asl-1.9.2.jar,
- lib/jackson-mapper-asl-1.9.2.jar
-Service-Component: OSGI-INF/*.xml
+Manifest-Version: 1.0
+Bundle-Activator: org.openhab.core.compat1x.internal.CompatibilityActiva
+ tor
+Bundle-ClassPath: .,lib/jackson-core-asl-1.9.2.jar,lib/jackson-mapper-as
+ l-1.9.2.jar

```

Comment:
this looks broken

Reply:
The question about new lines for bundle classpath attribute has already been aked into one of the PRs. Waiting for a response.

Reply:
Damn, I haven't pressed on "finish" to send the comment:
https://github.com/openhab/openhab2-addons/pull/2720#pullrequestreview-63960619